readme.md

```diff
@@ -0,0 +1,225 @@
+<p align = "center">Figure 6: The Connector and PCB Model in AppCSXCAD (openEMS)</p>
+
+The final output of the project - which is hopefully of use for the community - is a multilayer KiCAD footprint for the connector. Officially the KiCAD footprint editor does not support shapes on the internal metal layers. However, the footprint file can be modified with a text editor. Here shapes can be placed on the internal layers which works just fine. 
+
+In order to use the KiCAD connector Footprint simply assign it to a symbol in the schematic and place "Copper Zones" (see example board) on the internal layers of your application board. Make sure to follow the  [bridges and frames](https://community.aisler.net/t/bridges-and-frames/49) design rules from AISLER. Everything else is handled in the KiCAD footprint.
+
+<p align = "center"><img src="./figures/connector_kicad.png" alt="fig_connector_kicad" width="700"></p>
+<p align = "center">Figure 7: Multi layer Connector Footprint in KiCAD</p>
+
+Finally to test and measure the connector transition a 6 layer back-to-back test board was created in KiCAD. A total of three boards were manufactured by AISLER for around 20€. 
+
+<p align = "center"><img src="./figures/testboard_kicad3d.png" alt="fig_tesetboard_kicad" width="700"></p>
+<p align = "center">Figure 8: Test Fixture of the Rosenberger 32K242-40ML5 SMA Connector on  the AISLER 6LayerHD Stackup in KiCAD</p>
+
+After the fabrication of the PCB the SMA connectors were hand soldered on the test board as shown in Figure 9. 
+
+<p align = "center"><img src="./figures/connector_solder.jpg" alt="fig_connector_solder" width="700"></p>
+<p align = "center">Figure 9: Rosenberger Connector Solder joint</p>
+
+## 6. Simulation Results
+
+OpenEMS writes field data in a format suitable for ParaView. ParaView is a open software to visualize vector fields. Figure 9 shows a E-Field plot at the xz (y=0) plane (20GHz). One can clearly spot the transitions from the coaxial mode Mode (vertical field vectors) to the the microstrip mode. Also the removal of the GND plane right under the connector Pad is noticeable. 
+
+<p align = "center"><img src="./figures/efield_xz.png" alt="fig_efield"></p>
+<p align = "center">Figure 10: Test Fixture of the Rosenberger 32K242-40ML5 SMA Connector on  the AISLER 6LayerHD Stackup</p>
+
+Above the right end of the inner conductor there is some small radiation visible. The field lines bend from the solder joint up to the top ouf the connector outer body. This undesired effect is given by the connectors construction and can not be avoided.
+
+At the edge where the connector and the PCB meet the internal copper layers of the PCB and the connector outer body form a parasitic parallel plate waveguide along the z dimension. It is important to avoid the excitation of this waveguide to prevent excessive leakage at higher frequencies. The field results clearly show that there is no radiation at the connector to pcb interface.
```

Comment:
```suggestion
The edge where the connector and the PCB meet the internal copper layers of the PCB and the connector outer body form a parasitic parallel plate waveguide along the z dimension. It is important to avoid the excitation of this waveguide to prevent excessive leakage at higher frequencies. The field results clearly show that there is no radiation at the connector to pcb interface.
```